tutorials/independence.py

```diff
@@ -313,3 +313,134 @@
 # These tests runs like :ref:`any other test<general indep>`.
 #
 # .. _[1]: https://link.springer.com/article/10.1007/s10182-020-00378-1
+
+r"""
```

Comment:
This tutorial is not showing up on the website. Make sure that you are formatting it correctly. See other sections in the independence tutorial for help with this

Reply:
Tried reformatting, let me know if issue appears fixed